core/src/cats/ddl/creates/postgresql.sql

```diff
@@ -154,21 +154,21 @@ CREATE TABLE Media
 (
     MediaId           SERIAL      NOT NULL,
     VolumeName        TEXT        NOT NULL,
-    Slot              INTEGER     DEFAULT 0,
-    PoolId            INTEGER     DEFAULT 0,
+    Slot              BIGINT      DEFAULT 0,
+    PoolId            BIGINT      DEFAULT 0,
```

Comment:
About the non forced but still existing foreign key 

`poolid, mediatypeid, storageid, deviceid, locationid, scratchpoolid, recyclepoolid`

I wouldn't change them because if we have query comparing them, recent version maybe emit warning or error because of type mismatch and would require casting. Our systemtest are all green, but I'm afraid we don't cover all situtations.

About LabelType should have always been a smallint (choice 0,1) and now ibmansi is deprecated ?

Reply:
If we change something here, we should also adapt the content of 
`docs/manuals/source/DeveloperGuide/catalog.rst`

Reply:
I addressed all your comments. Please recheck!